Client/src/Components/product-reviews/ReviewList.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Review from "./review";
+
+// Creates a list of reviews. Maps in the list from ProductReviews to Review to create each review
+function ReviewList({ list }) {
+  const [listCounter, setList] = useState("Latest");
+  const listNewDateFormat = [...list];
+  list.forEach((review, index) => {
+    listNewDateFormat[index].date = new Date(review.date);
+  });
+  const helpfulList = [...listNewDateFormat].sort((a, b) => b.helpfulness - a.helpfulness);
+  const latestList = [...listNewDateFormat].sort((a, b) => b.date - a.date);
+
+  function onChange() {
+    const e = document.getElementById("test");
+    const selectedText = e.options[e.options.selectedIndex].text;
```

Comment:
If you pass in the event to onChange, you can access the selected text at event.target.value.
See docs on select element : https://reactjs.org/docs/forms.html